runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -211,7 +211,9 @@
     @Getter
     private static final String mpObj = mp + "object.";
     @Getter
-    private static final MetricRegistry metrics = new MetricRegistry();
+    private static MetricRegistry defaultMetrics = new MetricRegistry();
```

Comment:
You probably don't want it to be static. Different Runtimes should have different registries